src/providers/bitbucket.com.ts

```diff
@@ -1,7 +1,9 @@
 import { GitProvider } from '../gitProvider.ts';
+import { inversify, unmanaged } from '../../deps.ts';
 
+@inversify.injectable()
```

Comment:
should switch from `@inversify.injectable()` to just `@injectable()` - that was an oversight.